src/carousel/carousel.scss

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../text/index';
+@import '../button/button';
+
+@mixin iui-carousel {
+  @include iui-reset;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+}
+
+@mixin iui-carousel-slider {
+  display: flex;
+  position: relative;
+  overflow: hidden;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+
+  &-item {
+    position: absolute;
+    margin-left: auto;
+    margin-right: auto;
+    left: 0;
+    right: 0;
+    flex-shrink: 0;
+    width: 100%;
+    box-sizing: border-box;
+    @media (prefers-reduced-motion: no-preference) {
+      transition: transform $iui-speed-slow ease-in-out;
+    }
+
+    &.iui-previous {
+      transform: translateX(-100%);
+    }
+
+    &.iui-next {
+      transform: translateX(100%);
+    }
+
+    &.iui-current {
+      transform: translateX(0);
+    }
+
+    &-image {
+      width: 100%;
+      height: 100%;
+      background-position: center;
+      background-size: cover;
+    }
+
+    &-title {
+      position: absolute;
+      left: 0;
+      bottom: 0;
+      padding: $iui-baseline $iui-sm;
+      box-sizing: border-box;
+      width: 100%;
+      transition: background-color $iui-speed-fast ease;
+      @include themed {
+        background-color: rgba(0, 0, 0, t(iui-opacity-4));
+        color: t(iui-color-foreground-accessory);
+      }
+
+      &:hover {
+        @include themed {
+          background-color: rgba(0, 0, 0, t(iui-opacity-3));
+        }
+      }
+    }
+  }
+
+  + .iui-carousel-navigation {
+    @include themed {
+      border-top: $iui-xxs solid t(iui-color-background-4);
+    }
+  }
+}
+
+@mixin iui-carousel-navigation {
+  display: flex;
+  align-items: center;
+  height: $iui-baseline * 3;
+
+  &-dots {
+    @include iui-reset;
+    display: flex;
+    flex-grow: 1;
+    white-space: nowrap;
+    overflow: hidden;
+    justify-content: center;
+    align-items: center;
+    list-style: none;
+  }
+
+  &-dot {
+    @include iui-button;
+    @include iui-button-size($size: small);
+    background-color: transparent;
+    outline: none;
+
+    &:hover:not(.iui-invisible):not(.iui-active)::after {
```

Comment:
This is massive.. sure cannot be simplified?

Reply:
I believe I can remove `:not(.iui-invisible)` since `.iui-invisible`'s scale is at 0%.

Reply:
I was able to remove .iui-active too and reduce specificity from 0.2.1 to 0.1.1
Try it at your end. If it works same, commit that change.
It works, because we have iui-active::after rule below, which has same specificity.

Reply:
You are correct, pushed the change.